src/ext/components/model-parameter-list.pug

```diff
@@ -2,11 +2,11 @@ tr(data-bind="css:{'bg-danger':hasError, 'bg-warning':hasWarning, 'bg-info':hasI
   th
     label(data-bind='text:label')
     span(data-bind='visible:isRequired') *
-  td(colspan='2')
+  td
     div(style='height:25px')
       input.flow-textbox(type='text' placeholder='Search...' data-bind="value:searchTerm, valueUpdate:'afterkeydown'")
-    div(style='font-size:80%' data-bind='text:searchCaption')
-    div(style='margin-bottom:5px;height:300px;overflow:auto;border:1px dotted #ccc')
+    div(style='font-size:80%' data-bind='visible:navEnabled, text:searchCaption')
+    div(style='margin-bottom:5px;overflow:auto;border:1px dotted #ccc')
```

Comment:
I need to revisit this: overflow is not needed anymore if I remove `height`.
Previously, we were using both fixed height with overflow (and therefore potential vertical scrolling) + pagination: which is a complete UI design anti-pattern.
I removed the fixed height, and lower the max items from 100 to 25 per page: no scrolling, only pagination. But 25 may still be too much.
I'll probably lower to 15 or 20: any concern?

Reply:
This component is mainly used for multiple-columns selection.

Reply:
Decided to display only 10 items per page: makes the selector just reasonably high compared with other controls.